src/pages/docs/lexicon/image_card.md

```diff
@@ -0,0 +1,163 @@
+layout: "lexiconMigrationGuide"
+weight: 101
+
+clayComponentLink: "https://claycss.com/docs/components/cards.html"
+lexiconPattern: "Image Card"
+lexiconPatternLink: "https://lexicondesign.io/docs/patterns/cards.html"
+---
+
+<article id="before-after">
+
+### How does it look like now (7.0)
+
+<img class="img img-thumbnail" src="/images/lexiconMigration/image_card_old.png">
+
+### Changes for Lexicon 2.0 (7.1)
+
+<img class="img img-thumbnail" src="/images/lexiconMigration/image_card_new.png">
+
+Change | Reason
+--- | ---
+① The **icons spritemap** has changed | Fit with the new Lexicon icons
+② The **link style** now is black, blue on hover. Also the entire card has hover state | Fit with new links styles and better visual feedback
+③ The card no supports labels
+
+</article>
+
+<article id="steps">
+
+### How to apply it?
+
+_ | Action | Mandatory
+--- | --- | ---
+[1](#step-1) | Locate the code in your application | ✔
+[2](#step-2) | Use `clay:image-card` instead of the old `liferay-frontend:vertical-card` tag | ✔
+[3](#step-3) | Move the action items definition into a Display Context | ✘
+
+### [1] Locate the code in your application <a id="step-1"></a>
+
+Locate the code responsible for rendering the **image card** in your application. It should resemble something like the following snippet found in [journal/journal-web/view_entries.jsp](https://github.com/liferay/liferay-portal/blob/master/modules/apps/web-experience/journal/journal-web/src/main/resources/META-INF/resources/view_entries.jsp#L140):
+
+```text/html
+<liferay-frontend:vertical-card
+	actionJsp='<%= journalDisplayContext.isShowEditActions() ? "/article_action.jsp" : null %>'
+	actionJspServletContext="<%= application %>"
+	imageUrl="<%= HtmlUtil.escape(articleImageURL) %>"
+	resultRow="<%= row %>"
+	rowChecker="<%= articleSearchContainer.getRowChecker() %>"
+	title="<%= curArticle.getTitle(locale) %>"
+	url="<%= editURL %>"
+>
+	...
+</liferay-frontend:vertical-card>
+```
+
+### [2] Use `clay:image-card` instead of the old `liferay-frontend:vertical-card` tag <a id="step-2"></a>
+
+Start by adding the following imports to the imports section of your `init.jsp` file:
+
+```text/html
+// Import the clay tld file to be able to use the new tag
+<%@ taglib uri="http://liferay.com/tld/clay" prefix="clay" %>
+
+// Import the DropdownItem utility class to create the items model
+<%@ page import="com.liferay.frontend.taglib.clay.servlet.taglib.util.JSPDropdownItemList" %>
+```
+
+Don't forget to add the dependencies with to the `frontend-taglib-clay` and `fronteng.taglib.soy` module in your `build.gradle` file:
+
+```text/html
+provided group: "com.liferay", name: "com.liferay.frontend.taglib.soy", version: "1.0.0"
+
+provided project(":apps:foundation:frontend-taglib:frontend-taglib-clay")
+```
+
+- Url param is now href.
+- Text param is now title.
+- Icon is passed now trough icon param instead as a content with `horizontal-card-col` and `horizontal-card-col-icon`
```

Comment:
This doesn't work in the same way as `horizontal-card`. Here `icon` refers to the icon to render instead of an image if `imageSrc` is not passed.